tutorials/background_segmenter/code/camera.js

```diff
@@ -45,6 +45,9 @@ export default class Camera {
             };
             this.stream = await navigator.mediaDevices.getUserMedia(constraints);
             this.videoElement.srcObject = this.stream;
+            if(/^((?!chrome|android).)*safari/i.test(navigator.userAgent)) {
+                this.videoElement.play();
```

Comment:
This promise should be awaited and the API can be called regardless of browser (so no need for Safari check).

Thanks!

Reply:
Makes sense. TIL ```play()``` is a promise. Just pushed those changes.

Reply:
And yea thank you. This repo was massively helpful in what I'm making.